packages/components/src/text-control-with-affixes/example.md

```diff
@@ -0,0 +1,31 @@
+```jsx
+import { TextControlWithAffixes } from '@woocommerce/components';
+
+const MyTextControlWithAffixes = withState( {
+	first: '',
+    second: '',
```

Comment:
super nitpick– mix of tabs and spaces here. AFAIK we don't have a standard in these md files, but it looks like we use tabs in the others.

Reply:
Fixed in cc92f19.